src/CarrierInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Meanbee\Royalmail;
+
+interface CarrierInterface
```

Comment:
Good use of new interface, this will allow us to add more methods if we need, and makes the library more SOLID.